### PR TITLE
Adding documentation to add asset and scanning in Ralph quick start

### PR DESCRIPTION
**Pull request by szok**
Beautifull quickstart.

### CODE REVIEW

File: doc/ralph_quick_start.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Let's learn Ralph by example. Throughout this tutorial, we’ll walk you through:
+
+- adding asset information, licenses and support
+- scanning asset details and configuration
+- access billing information
+
+
+Add new asset
+----------------
+Open the Assets tabs with you can manage provides advanced Asset management system.
```
Comment (vi4m) [inline doc/ralph_quick_start.rst]: ?

File: doc/quickstart.rst [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+
+
+Assets
+^^^^^^
+
+Searching and Filtering of Assets
+---------------------------------
+
+Let's start with the main screen. Here you find all your hardware assets which
+are in the database. Use the left column for filtering of the assets.
```
Comment (deejay1) [inline doc/quickstart.rst]: maybe "fitlering the assets" will be enough?

File: doc/quickstart.rst [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+There are two types of assets -- devices and parts. A device could be a blade
+server, and a part is a component of this server, for example memory or hard
+disk drive. A part can be assigned to a single device at a time. You can move
+parts from one device to another when you need it.
+
+
+Adding Assets
+-------------
+
+If you want add data center asset you need to fill a few required fields.
```
Comment (deejay1) [inline doc/quickstart.rst]: *add a data center asset* ?

File: doc/quickstart.rst [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+Look at the screen below (you can zoom-in the image in your browser if you
+want) The fields in bold are required. Rest of fields are optional.
```
Comment (deejay1) [inline doc/quickstart.rst]: maybe emphasize **bold** for better readability?

File: doc/quickstart.rst [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+
+.. image:: _static/add-data-center-asset-form.png
+    :scale: 30%
+
+Let's describe each of the fields:
+
+- **Basic info**:
+    - **Type** - a read only field for data center (example: )
```
Comment (deejay1) [inline doc/quickstart.rst]: *(example: )* what? similar below  IMHO it's better leave it out alltogether if it's empty for now

File: doc/quickstart.rst [hunk 6]
```diff
@@ -0,0 +1,294 @@
...
+    - **Status** - an asset's lifetime indicator. Newly bought assets has
+      status "new". You can change it as required according to your own work
+      flow (example: )
+    - **Task url** - url to task in ticket system (example: )
+    - **Additional remarks** - field for various data (example: )
+    - **Service name** - service name to which this asset belongs (example: )
+    - **Property of** - company to which this asset belongs (example: )
+    - **Hostname** - read-only field telling the name of host (from Ralph-core
+      device) (example: )
+    - **Service catalog** - points to ``service catalog`` which asset belongs
```
Comment (deejay1) [inline doc/quickstart.rst]: *service from the ``service catalog``*? as it doesn't point to the whole catalogue ;)

File: doc/quickstart.rst [hunk 7]
```diff
@@ -0,0 +1,294 @@
...
+      ``depreciation end date`` value
+- **User info**:
+    - **User** - points to user of this asset
+    - **Owner** - points to a owner of this asset
+- **Aditional info**:
+    - **Data center** - points to a data center which contains this asset
+    - **Server room** - points to a server room which contains this asset
+    - **Rack** - points to a rack which contains this asset
+    - **Position** - points to position in rack from 0 to 'max u'
+    - **Orientation** - a side of rack, depends on ``position``
```
Comment (deejay1) [inline doc/quickstart.rst]: IMHO drop the initial *a*

File: doc/quickstart.rst [hunk 8]
```diff
@@ -0,0 +1,294 @@
...
+.. image:: _static//bulk-2.png
+    :scale: 55%
+
+On the next screen you can edit those records all at once by changing the
+appropriate fields. When you fill one field with the desired value, you can
+propagate this value to all records by clicking on the "plus" mark near the
+current cell.
+
+
+Work Flow and Statues
```
Comment (deejay1) [inline doc/quickstart.rst]: Work Flow and *Statuses*

File: doc/quickstart.rst [hunk 9]
```diff
@@ -0,0 +1,294 @@
...
+
+To add a license, click the "Add support" option from the top of submenu.
+
+.. image:: _static/add_licence.png
+    :scale: 75%
+
+
+
+Licence fields are split into 2 section: **Basic info** and **Financial info**.
+**Financial info** contains very important field, **Number of purchased items**.
```
Comment (deejay1) [inline doc/quickstart.rst]: containes *a* very important field?

File: doc/quickstart.rst [hunk 10]
```diff
@@ -0,0 +1,294 @@
...
+This field ability to store Multi-Seat licence.
```
Comment (deejay1) [inline doc/quickstart.rst]: *has the ability*, otherwise it sounds weird

File: doc/quickstart.rst [hunk 11]
```diff
@@ -0,0 +1,294 @@
...
+
+
+Relations
+---------
+
+Licenses may be related to the relationship with the user or device.
```
Comment (deejay1) [inline doc/quickstart.rst]: erm? Maybe *Licences ,may have a relationship with a user or device* or *Licenses may be related to a user or device*

File: doc/quickstart.rst [hunk 12]
```diff
@@ -0,0 +1,294 @@
...
+In asset and user form, during the search are shown only unassigned license,
```
Comment (deejay1) [inline doc/quickstart.rst]: *only unassigned licenses are shown* - also please decide if you use licence or license in the entire document

File: doc/quickstart.rst [hunk 13]
```diff
@@ -0,0 +1,294 @@
...
+
+To add a support, click the "Add support" option from the top of submenu.
+
+.. image:: _static/add_support.png
+    :scale: 75%
+
+
+Scrooge
+^^^^^^^
+Scrooge module brings billing functionality to Ralph. Using Ralph Scrooge, you can retrieve information on the use of resources from many other systems and charge other services. Scrooge generates flexible and accurate cost reports and let you refer to historical usages and costs.
```
Comment (deejay1) [inline doc/quickstart.rst]: *lets you refer*

File: doc/quickstart.rst (hunk 2 above)
Comment (szok) [inline doc/quickstart.rst]: ok

File: doc/quickstart.rst (hunk 3 above)
Comment (szok) [inline doc/quickstart.rst]: Fact, ok

File: doc/quickstart.rst (hunk 5 above)
Comment (szok) [inline doc/quickstart.rst]: Empty because I'm waiting for fixtures then I will add those names which generate demo_data

**Comment by vi4m** [comment]
@szok please fix deejay1 comments, we will merge it and work on it further.

**Comment by vi4m** [comment]
LGTM

**Pull request merged**